README.md

```diff
@@ -517,3 +518,16 @@ Commands:
   worklog list:        Prints the worklog data for given issue
 
 ```
+
+#### Setting up TAB completion
```

Comment:
This wont match your #tab-completion anchor.  GH Markdown is not that clever.  So you can change the TOC to `#setting-up-tab-completion` or change the topic to `#### TAB completion`.

Reply:
Good catch. Fixed!